CancelablePromise.js

```diff
@@ -65,8 +65,11 @@ export default class CancelablePromise {
     return this.then(undefined, error);
   }
 
-  cancel() {
+  cancel(error) {
```

Comment:
could we make this 'errorCallback' instead as I think it conveys better the fact that it's a function